### PR TITLE
V3 launch

### PR DESCRIPTION
**Pull request by msub2**
- Removed beta overrides for all but one of the trial spaces
- Bumped all public package versions to 3.0.0
- Adjust a couple of deprecated WLE calls
- Adjust retry count tracking to account for multiple banners at once properly
- Update tests

### CODE REVIEW

**Comment by msub2** [comment]
Fixed the issue with image tests failing but the navigation one is still timing out. Looking into it

File: wonderland/src/index.js [hunk 1]
```diff
@@ -95,25 +93,14 @@ export class ZestyBanner extends Component {
...
-          }
         });
-    } else {
-      this.startLoading();
-      if (isBeta) {
-        setInterval(this.startLoading.bind(this), AD_REFRESH_INTERVAL);
-      }
     }
+    this.startLoading();
+    setInterval(this.startLoading.bind(this), AD_REFRESH_INTERVAL);
```
Comment (limbofeather) [inline wonderland/src/index.js:103]: Can we drop the refresh interval to 15 sec while we're at it?

Comment (limbofeather) [reply wonderland/src/index.js:103]: Might also be better to import this refresh interval globally from `networking.js`

File: utils/networking.js [hunk 2]
```diff
@@ -48,40 +48,33 @@ const initPrebid = (adUnitId, format) => {
...
-  { id: '14dccdbe-18b7-40d0-93d8-c104fd9486e8', format: 'medium-rectangle' },
-  { id: 'a8e0496f-034d-4cea-ba5f-653bba4fba39', format: 'billboard' },
-  { id: 'a181cc07-fda7-462e-adba-0fd8abf0af24', format: 'billboard' },
+const unitOverrides = [
+  { id: '4902864a-5531-496b-8d4d-ec7b9849e8e1', format: 'medium-rectangle', oldFormat: 'tall', absoluteWidth: 0.75, absoluteHeight: .625 },
 ];
 
-const getV3BetaUnitInfo = (adUnitId) => {
-  return betaUnits.find(unit => unit.id === adUnitId) || {};
+const getOverrideUnitInfo = (adUnitId) => {
```
Comment (limbofeather) [inline utils/networking.js:55]: What exactly are we overriding if we're removing the list of beta ad units?

File: utils/networking.js [hunk 3]
```diff
@@ -62,6 +62,10 @@ const getDefaultBanner = (format, style, shouldOverride, overrideFormat) => {
 }
 
 const fetchCampaignAd = async (adUnitId, format = 'tall', style = 'standard') => {
+  if (['tall', 'wide', 'square'].includes(format)) {
+    console.warn(`The old Zesty banner formats (tall, wide, and square) are being deprecated and will be removed in a future version. Please update to one of the new IAB formats (mobile-phone-interstitial, billboard, and medium-rectangle).`);
```
Comment (limbofeather) [inline utils/networking.js:66]: Might be helpful to link them to docs from here?

**Review by limbofeather — APPROVED** [review]
Looking 🍏

**Pull request merged**